src/main/java/uk/gov/hmcts/divorce/print/BulkPrintService.java

```diff
@@ -48,4 +62,21 @@ public UUID print(Print print) {
                 )))
             .letterId;
     }
+
+    private byte[] getDocumentBytes(final String docUrl, final String authToken)  {
+        final var userAuth = request.getHeader(AUTHORIZATION);
+        final var userDetails = idamService.retrieveUser(userAuth).getUserDetails();
+        var resourceResponseEntity = documentManagementClient.downloadBinary(
+            userAuth,
+            authToken,
+            String.join(",", userDetails.getRoles()),
+            userDetails.getId(),
+            FilenameUtils.getName(docUrl)
+        );
+        try {
+            return resourceResponseEntity.getBody().getInputStream().readAllBytes();
+        } catch (Exception e) {
+            throw new BulkPrintException("Doc url " + docUrl, e);
```

Comment:
it's Document download exception rather than bulk print

Reply:
Changed to DocumentDownload exception.